lib/operations/close.js

```diff
@@ -23,13 +23,22 @@ class CloseOperation extends OperationBase {
       client.removeAllListeners('close');
       callback(err, null);
     };
+    const mongocryptdClientClose = err => {
+      const mongocryptdClient = client.s.mongocryptdClient;
```

Comment:
Not for this PR, but would it make sense to make `mongocryptdClient` a property on `client`?

Reply:
IMO it's not worth exposing as part of public API.

Reply:
I agree, if possible I'd like to fully remove all of this from the driver and keep it encapsulated in `mongodb-client-encryption`. It may not be possible though.